pkg/util/allocationfilterutil/v2/parser.go

```diff
@@ -48,8 +51,9 @@ import (
 // <filter-field-2> ::= 'label' | 'annotation'
 //
 // <filter-field-1> ::= 'cluster' | 'node' | 'namespace'
-//                    | 'controllerName' | 'controllerKind'
-//                    | 'container' | 'pod' | 'services'
+//
+//	| 'controllerName' | 'controllerKind'
+//	| 'container' | 'pod' | 'services'
```

Comment:
Please revert these changes, the spacing here is important.

Reply:
GitHub didn't grab the whole section, but I meant to grab the whole grammar section in this comment.

Reply:
You got it!

Reply:
Should be good.